content/call-for-paper.md

```diff
@@ -1,25 +1,27 @@
 ---
 title: ارائه و ارسال مقاله
-description: "حمایت از جشن آزادی نرم‌افزار ۱۴۰۴"
+description: 'حمایت از جشن آزادی نرم‌افزار ۱۴۰۴'
 ---
 
 {{< banner src="/image/call-for-paper.jpg" alt="call for papers" >}}
 
-
 ## میکروفون دست شماست!
+
 می‌خواهید درمورد موضوعی ارائه داشته باشید؟ یا محصولی دارید که مایلید معرفی‌ش کنید؟ این رویداد فرصت خوبیه برای دیده شدن. تا ۲۰ شهریور فرصت دارید تا از طریق لینک زیر و مطالعه شرایط، موضوع پیشنهادی‌تون رو ارسال کنید.
 
 - [فرم ارائه/ارسال مقاله](https://survey.porsline.ir/s/F6i46onN)
 
-*مهم نیست چقدر تجربه دارید. بدون ترس موضوع‌تون رو ارسال کنید، ما به شما کمک می‌کنیم تا بهترین ارائه رو داشته باشید.*
+_مهم نیست چقدر تجربه دارید. بدون ترس موضوع‌تون رو ارسال کنید، ما به شما کمک می‌کنیم تا بهترین ارائه رو داشته باشید._
```

Comment:
what's the difference? asking just out of curiosity :)) i think they're both converted to italic